docs/best_practices/tool_xml.rst

```diff
@@ -117,7 +117,7 @@ It's convenient to do something like:
 
 .. code:: console
 
-    ln -sfn $input\_fasta tmp.fa;
+    ln -sfn "${input}"\_fasta tmp.fa;
```

Comment:
That would be simpler (although underscores are common in parameter names so having this in the example is quite educational)